napari/_qt/_qapp_model/qactions/_window.py

```diff
@@ -0,0 +1,70 @@
+"""Defines window menu actions."""
+
+from typing import List
+
+from app_model.types import Action
+
+from ...._app_model.constants import CommandId, MenuGroup, MenuId
+from ....utils.translations import trans
+from ...qt_main_window import Window
+
+
+def _toggle_console(window: Window):
+    window._qt_viewer.dockConsole.setVisible(
+        not window._qt_viewer.dockConsole.isVisible()
+    )
+
+
+def _toggle_layer_controls(window: Window):
+    window._qt_viewer.dockLayerControls.setVisible(
+        not window._qt_viewer.dockLayerControls.isVisible()
+    )
+
+
+def _toggle_layer_list(window: Window):
+    window._qt_viewer.dockLayerList.setVisible(
+        not window._qt_viewer.dockLayerList.isVisible()
+    )
+
+
+WINDOW_ACTIONS: List[Action] = [
```

Comment:
I would maybe call them `Q_WINDOW_ACTIONS` as they're in the `qactions` folder.

Reply:
I've commented on the other related PRs so they are consistent